shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -23,25 +32,69 @@ class FormatMoneyUseCase @Inject constructor(
     private val withoutDecimalFormatter = DecimalFormat("###,###", DecimalFormatSymbols(locale))
     private val withDecimalFormatter = DecimalFormat("###,###.00", DecimalFormatSymbols(locale))
     private val shortenAmountFormatter = DecimalFormat("###,###.##", DecimalFormatSymbols(locale))
+    private val cryptoFormatter =
+        DecimalFormat("###,###,##0.${"0".repeat(9)}", DecimalFormatSymbols(locale))
```

Comment:
Can you try fixing the formatter by `"#".repeat(9)`

Reply:
Done